scripts/tools/zap/tests/outputs/all-clusters-app/app-templates/endpoint_config.h

```diff
@@ -26,7 +26,7 @@
 // Default values for the attributes longer than a pointer,
 // in a form of a binary blob
 // Separate block is generated for big-endian and little-endian cases.
-#if BIGENDIAN_CPU
+#if CHIP_CONFIG_BIG_ENDIAN_TARGET
```

Comment:
This is supposed to be defined in CHIPConfig.h, right?

Reply:
Hmm...  Good catch.  I thought `CHIPConfig.h` pulled in `CHIPDeviceBuildConfig.h`, but looks like no.  I need to move these bits to `CHIPBuildConfig.h` instead.